tests/core/pyspec/eth2spec/test/helpers/rewards.py

```diff
@@ -61,6 +62,17 @@ def run_attestation_component_deltas(spec, state, component_delta_fn, matching_a
                 assert penalties[index] == 0
 
 
+def set_some_new_deposits(spec, state, rng):
+    num_validators = len(state.validators)
+    # last 10th of validators are new deposits
+    for i in range(len(state.validators))[0:num_validators // 10]:
```

Comment:
`[0:num_validators // 10]` are the first 10 validators, do you mean:

```suggestion
    for i in range(num_validators)[-num_validators // 10:]:
```